### PR TITLE
Adding coorganizers in event application form

### PR DESCRIPTION
**Pull request by aniav**
Added:
* adding coorganizers to the event application
* e-mail confirming an application was submitted and we have received it

### CODE REVIEW

File: templates/organize/form/step3_organizers.html [hunk 1]
```diff
@@ -36,98 +37,159 @@
...
+                            var inputs = formTemplate.content.querySelectorAll("input");
+                            inputs.forEach(function(element) {
+                                element.name = element.name.replace('__prefix__', formCount);
+                            })
+                            var form = document.importNode(formTemplate.content, true);
+                            $('#coorganizers').append(form);
+                            formCount++;
+                            $('#id_organizers-TOTAL_FORMS').val(formCount);
+                        });
+                    </script>
```
Comment (olasitarska) [inline templates/organize/form/step3_organizers.html]: Can we move javascript from inline into [this file](https://github.com/DjangoGirls/djangogirls/blob/master/static/source/js/organize.js)?

File: templates/organize/form/step3_organizers.html [hunk 2]
```diff
@@ -36,98 +37,159 @@
...
                                         {% endfor %}
                                     </ul>
                                 {% endif %}
                             </li>
+                            {% endwith %}
                         </ul>
                     </fieldset>
 
-                    <!--
                     {# Static mock up of HTML used for this section #}
```
Comment (olasitarska) [inline templates/organize/form/step3_organizers.html]: Outdated comment

File: templates/organize/form/step3_organizers.html (hunk 1 above)
Comment (aniav) [inline templates/organize/form/step3_organizers.html]: Yeah, this code is totally not finished :)

**Comment by aniav** [comment]
This is merged.

**Pull request closed without merging**